contracts/mocks/Create2Impl.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.5.0;
+
+import "../utils/Create2.sol";
+
+contract Create2Impl {
```

Comment:
I would like to see a test that deploys with `Create2` from within Solidity. It would look like `Create2.deploy(salt, type(ERC20).creationcode)`.

Reply:
How is this different from the `deploy(bytes32 salt, bytes memory code) public` test?